app/src/form/support/FieldGroupHeader.js

```diff
@@ -7,9 +7,17 @@ const FieldGroupHeader = ({ index, name = '', description = '' }) => {
   return (
     <View>
       <View style={styles.fieldGroupHeader}>
-        <Text style={styles.fieldGroupName}>{!isNaN(index) ? `${index + 1}. ${name}` : name}</Text>
+        <Text style={styles.fieldGroupName} testID="text-name">
+          {!isNaN(index) ? `${index + 1}. ${name}` : name}
```

Comment:
why we have `!isNaN(index)` ? this means field group headers sometimes doesn't have index?

Reply:
This means the Header component has an option to use the index or not, and `isNaN` for this case to handle the index is not a number.

But I think we can change the conditional statement like this
```
index ? `${parseInt(index) + 1}. ${name}` : name
```

is this Okay for you mas @dedenbangkit?

Reply:
wait, I think an index should always exist and is a number if we look based on this use case
https://github.com/akvo/nmis-mobile/blob/main/app/src/form/components/QuestionGroup.js#L9

https://github.com/akvo/nmis-mobile/blob/main/app/src/form/FormContainer.js#L70C2

https://github.com/akvo/nmis-mobile/blob/main/app/src/form/lib/index.js#L71

Reply:
Please make it **strict**. Array will always have index.